demo/agw/ScrolledThumbnail.py

```diff
@@ -0,0 +1,640 @@
+        self.setpopup = wx.CheckBox(self.panel, -1, "Set popup menu on thumbs")
+        self.setgpopup = wx.CheckBox(self.panel, -1, "Set global popup menu")
+        self.enabletooltip = wx.CheckBox(self.panel, -1, "Enable thumb tooltips")
+        self.textzoom = wx.TextCtrl(self.panel, -1, "1.4")
+        self.zoombutton = wx.Button(self.panel, -1, "Set zoom factor")
+        self.textthumbwidth = wx.TextCtrl(self.panel, -1, "96")
+        self.textthumbheight = wx.TextCtrl(self.panel, -1, "80")
+        self.thumbsizebutton = wx.Button(self.panel, -1, "Set thumbnail size (WxH)")
+        self.fontbutton = wx.Button(self.panel, -1, "Set caption font")
+        self.colourbutton = wx.Button(self.panel, -1, "Set selection colour")
+
+        self.radios = [self.radiostyle1, self.radiostyle2, self.radiostyle3,
+                       self.radiostyle4]
+        self.thumbstyles = ["THUMB_OUTLINE_NONE", "THUMB_OUTLINE_FULL", "THUMB_OUTLINE_RECT",
+                            "THUMB_OUTLINE_IMAGE"]
+
+        self.SetProperties()
+        self.DoLayout()
+
+        self.panel.SetSizer(sizer)
+        sizer.Layout()
+
+        self.Bind(wx.EVT_RADIOBUTTON, self.OnChangeOutline, self.radiostyle1)
+        self.Bind(wx.EVT_RADIOBUTTON, self.OnChangeOutline, self.radiostyle2)
+        self.Bind(wx.EVT_RADIOBUTTON, self.OnChangeOutline, self.radiostyle3)
+        self.Bind(wx.EVT_RADIOBUTTON, self.OnChangeOutline, self.radiostyle4)
+        self.Bind(wx.EVT_CHECKBOX, self.OnHighlight, self.highlight)
+        self.Bind(wx.EVT_CHECKBOX, self.OnShowFiles, self.showfiles)
+        self.Bind(wx.EVT_CHECKBOX, self.OnEnableDragging, self.enabledragging)
+        self.Bind(wx.EVT_CHECKBOX, self.OnSetPopup, self.setpopup)
+        self.Bind(wx.EVT_CHECKBOX, self.OnSetGlobalPopup, self.setgpopup)
+        self.Bind(wx.EVT_CHECKBOX, self.OnEnableToolTips, self.enabletooltip)
+        self.Bind(wx.EVT_BUTTON, self.OnSetZoom, self.zoombutton)
+        self.Bind(wx.EVT_BUTTON, self.OnSetThumbSize, self.thumbsizebutton)
+        self.Bind(wx.EVT_BUTTON, self.OnSetFont, self.fontbutton)
+        self.Bind(wx.EVT_BUTTON, self.OnSetColour, self.colourbutton)
+        self.Bind(wx.EVT_BUTTON, self.OnSetDirectory, self.dirbutton)
+
+        self.scroll.Bind(EVT_THUMBNAILS_SEL_CHANGED, self.OnSelChanged)
+        self.scroll.Bind(EVT_THUMBNAILS_POINTED, self.OnPointed)
+        self.scroll.Bind(EVT_THUMBNAILS_DCLICK, self.OnDClick)
+
+        splitter.SplitVertically(self.scroll, self.panel, 400)
+
+        splitter.SetMinimumPaneSize(140)
+        self.SetMinSize((1000, 1000))
+        self.CenterOnScreen()
+
+    def SetProperties(self):
+
+        self.radiostyle4.SetValue(1)
+        self.showfiles.SetValue(1)
+        boldFont = wx.Font(8, wx.DEFAULT, wx.NORMAL, wx.BOLD, 0, "")
+        self.zoombutton.SetFont(boldFont)
+        self.fontbutton.SetFont(boldFont)
+        self.dirbutton.SetFont(boldFont)
+        self.colourbutton.SetFont(boldFont)
```

Comment:
An 8 point font looks very odd on OSX. I expect that leaving the font alone on these buttons would be better.

Reply:
10 or 12 is norm on other platforms. 14 Should be ok on all, but might look a bit oversized for the default size.

Reply:
Which is why I suggested removing the font changes, because the default for the platform should be best for the demo and would not need a lot of thought and tweaking to try to find something that works best everywhere for what is, after all, just a normal button.

Reply:
Removed all this font code.  Default looks OK to me.